### ISSUE TITLE
vagrant_ip has conflict with MikroTik router

### ISSUE THREAD

**Issue by Niklan** [geerlingguy/drupal-vm#1565]
### Issue Type

 - Question

### Summary

Is there any reason why for variable `vagrant_ip` the ip `192.168.88.88` is used by default? I bought yesterday MikroTik router which binds 192.168.88.* to it's own purposes. So this cause the problem with starting vagrant:

```
The specified host network collides with a non-hostonly network!
This will cause your specified IP to be inaccessible. Please change                                                                                                                                                 
the IP or name of your host only network so that it no longer matches that of                                                                                                                                       
a bridged or non-hostonly network.       
```

I just want you to know it.

**Comment by thom8** [comment]
@Niklan there's a PR open which could fix this -- https://github.com/geerlingguy/drupal-vm/pull/1559

**Comment by geerlingguy** [comment]
I have no idea why I originally chose 88.88, other than a general liking for the number 8... so it kind of just stuck.

Note that until anything is changed in Drupal VM's default IP, changing the IP address is easy with `vagrant_ip` or `vagrant_public_ip`; or you can use auto networking if you have that plugin by setting `0.0.0.0`, or even `dhcp` if you check out how it's done in #1559 .

**Comment by Niklan** [comment]
@geerlingguy yes, this is not problem at all. I just changed the IP on 192.168.1.88 (was binded to previous asus router, so now it's free to use in my network) and all works fine. 

I just wrote it for your information, maybe it will be helpful in the future, so issue now can be closed. Maybe some of IP in 192.168.* is safe to use on all routers and can be set by default? I don't know. I also used before D-Link, it also bind 192.168.1.* as asus router.
